pod.go

```diff
@@ -235,6 +235,13 @@ type PodStatus struct {
 	ContainersStatus []ContainerStatus
 }
 
+// ShimInfo gathers information needed by a shim, returned from a proxy.
+type ShimInfo struct {
+	Token string
+	IP    string
+	Port  string
+}
```

Comment:
One Pod could have many shims associated to it. `IP` and `Port` make sense, but I don't think `Token` does.